packages/fxa-auth-server/docs/swagger/descriptions.ts

```diff
@@ -0,0 +1,141 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import dedent from 'dedent';
+
+const DESCRIPTIONS = {
+  accessToken:
+    "An OAuth access token that the client can use to access data associated with the user's account.",
+  accessType:
+    'If specified, a value of `offline` will cause the connecting client to be granted a refresh token alongside its access token.',
+  acrValues:
+    'A space-separated list of ACR values specifying acceptable levels of user authentication. Specifying `AAL2` will ensure that the user has been authenticated with 2FA before authorizing the requested grant.',
+  authAt: 'Authentication time for the session (seconds since epoch).',
+  authAt2:
```

Comment:
There were two descriptions for `authAt`, do we want to combine them into one? Perhaps - `The timestamp for the session at which the user last authenticated to FxA, in seconds since the epoch`?

Reply:
WFM 👍